content/200-orm/200-prisma-client/500-deployment/301-edge/100-overview.mdx

```diff
@@ -0,0 +1,57 @@
+
+| Provider / Product      | Supported natively with Prisma ORM                      | Supported with Prisma Accelerate |
+| ----------------------- | ------------------------------------------------------- | -------------------------------- |
+| Vercel Edge Functions   | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Vercel Edge Middleware  | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Cloudflare Workers      | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Cloudflare Pages        | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Deno Deploy             | [Not yet](https://github.com/prisma/prisma/issues/2452) | ✅                               |
+| Netlify Edge Functions  | Not yet                                                 | ✅                               |
+| Supabase Edge Functions | Not yet                                                 | ✅                               |
+
+Deploying edge functions that use Prisma ORM on Cloudflare and Vercel is currently in [Preview](/orm/more/releases#preview).
+
+## Edge-compatibility of database drivers
+
+### Why are there limitations around database drivers in edge functions?
+
+Edge functions typically don't use the standard Node.js runtime. For example, Vercel Edge Functions and Cloudflare Workers are running code in [V8 isolates](https://v8docs.nodesource.com/node-0.8/d5/dda/classv8_1_1_isolate.html). Deno Deploy is using the [Deno](https://deno.com/) JavaScript runtime. As a consequence, these edge functions only have access to a small subset of the standard Node.js APIs and also have constrained computing resources (CPU and memory).
+
+In particular, the constraint of not being able to freely open TCP connections makes it difficult to talk to a traditional database from an edge function. While Cloudflare has introduced a [`connect()`](https://developers.cloudflare.com/workers/runtime-apis/tcp-sockets/) API that enables limited TCP connections, this still only enables database access using specific database drivers that are compatible with that API.
+
+> **Note**: [Prisma Accelerate](/accelerate) enables you to access _any_ database from _any_ edge function provider. No edge-compatible driver is necessary.
+
+### Which database drivers are edge-compatible?
+
+Here is an overview of the different database drivers and their compatibility with different edge function offerings:
+
+- [Neon Serverless](https://neon.tech/docs/serverless/serverless-driver) uses HTTP to access the database
+- [PlanetScale Serverless](https://planetscale.com/docs/tutorials/planetscale-serverless-driver) uses HTTP to access the database
+- [`node-postgres`](https://node-postgres.com/) (`pg`) uses Cloudflare's `connect()` (TCP) to access the database
```

Comment:
Missing Turso?

Reply:
Good point! Let's I pinged Jan already in Notion to get his thoughts on whether this should be included in the docs.

<img width="232" alt="image" src="https://github.com/prisma/docs/assets/4058327/7d7ddc6f-72d7-45bd-b6e6-fcfcc5ab6a46">

Reply:
Added Turso to the list ✅